model-optimizer/extensions/middle/ONNXResize11ToInterpolateV4.py

```diff
@@ -0,0 +1,166 @@
+                                           {1: int64_array([begin_dim]),
+                                            2: int64_array([end_dim]),
+                                            3: int64_array([1])},
+                                           {'name': resize_name + '/StridedSlice_',
+                                            'begin_mask': int64_array([1]),
+                                            'end_mask': int64_array([1]),
+                                            'new_axis_mask': int64_array([0]),
+                                            'shrink_axis_mask': int64_array([0]),
+                                            'ellipsis_mask': int64_array([0])})
+    scales_ss = create_op_with_const_inputs(graph, StridedSlice,
+                                            {1: int64_array([begin_dim]),
+                                             2: int64_array([end_dim]),
+                                             3: int64_array([1])},
+                                            {'name': resize_name + '/StridedSlice_',
+                                             'begin_mask': int64_array([1]),
+                                             'end_mask': int64_array([1]),
+                                             'new_axis_mask': int64_array([0]),
+                                             'shrink_axis_mask': int64_array([0]),
+                                             'ellipsis_mask': int64_array([0])})
+    axes_node = Const(graph,
+                      {'name': resize_name + '/axis_',
+                       'value': int64_array(np.arange(begin_dim, end_dim))}).create_node()
+
+    shape_calculation_mode = 'scales' if num_of_inputs == 3 else 'sizes'
+
+    interpolate_node = Interpolate(graph, {'version': 'opset4',
+                                           'mode': convert_mode(resize.mode),
+                                           'coordinate_transformation_mode': resize.coordinate_transformation_mode,
+                                           'cube_coeff': resize.cube_coeff,
+                                           'nearest_mode': resize.nearest_mode,
+                                           'pads_begin': int64_array([0]),
+                                           'pads_end': int64_array([0]),
+                                           'antialias': 0,
+                                           'shape_calculation_mode': shape_calculation_mode,
+                                           'in_ports_count': 4}).create_node()
+
+    axes_node.out_port(0).connect(interpolate_node.in_port(3))
+    shape_of = Shape(graph, {'name': resize_name + '/ShapeOf_'}).create_node()
+
+    if num_of_inputs == 3:
+        cast_shape_to_float = Cast(graph, {'dst_type': np.float32}).create_node()
+        mul_node = Mul(graph, {'name': resize_name + '/Mul_'}).create_node()
+        shape_of.out_port(0).connect(cast_shape_to_float.in_port(0))
+        cast_shape_to_float.out_port(0).connect(mul_node.in_port(0))
```

Comment:
I don't see adding 'epsilon' before converting to int. I think that we should add it here to avoid issues like 2.33333333 * 3 = 6.9999999 instead of 7

Reply:
Fixed.